app/controllers/claims/support/claims_controller.rb

```diff
@@ -11,7 +11,10 @@ def show; end
   def download_csv
     csv_data = Claims::GenerateClaimsCsv.call
 
-    send_data csv_data, filename: "claims.csv", type: "text/csv", disposition: "attachment"
+    today = Time.zone.today.strftime("%Y%m%d")
```

Comment:
I think you can just use `Date.current` here and should give you the same results https://apidock.com/rails/v3.2.1/Date/current/class
```suggestion
    today = Date.current.strftime("%Y%m%d")
```